policyengine_us/variables/gov/states/ri/tax/income/credits/property_tax/ri_property_tax_credit_eligible.py

```diff
@@ -23,6 +23,6 @@ def formula(tax_unit, period, parameters):
         is_disabled = head_is_disabled | spouse_is_disabled
         household_income = tax_unit("ri_property_tax_household_income", period)
         household_income_eligible = (
```

Comment:
do we need this? i don't see it in the law. i think we can leave this to the credit amount computation.

Reply:
We do because the percentage does not limit the credit to 0. The rate determines the percentage of income that reduces the amount of applicable rent. If the rate is 0 the total rent is applied but the credit amount is no reduced to 0.